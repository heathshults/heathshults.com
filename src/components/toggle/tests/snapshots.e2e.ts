import { snapIt } from '../../../test';

const component = 'toggle';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<hs-toggle>default</hs-toggle>');
    snap('colours based on type', '<hs-toggle type="success">default</hs-toggle>');
    snap('should be toggle on when prop is present', '<hs-toggle toggled type="success">default</hs-toggle>');
  });
});
