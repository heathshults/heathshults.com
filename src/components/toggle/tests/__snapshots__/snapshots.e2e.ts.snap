// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toggle renders colours based on type 1`] = `"<hs-toggle type=\\"success\\" class=\\"hydrated\\"><label class=\\"hs-toggle hs-toggle--success\\"><input type=\\"checkbox\\" aria-checked=\\"false\\"><div class=\\"hs-toggle__track\\"><div class=\\"hs-toggle__handle\\"></div></div>default</label></hs-toggle>"`;

exports[`toggle renders renders correctly with no props 1`] = `"<hs-toggle class=\\"hydrated\\"><label class=\\"hs-toggle\\"><input type=\\"checkbox\\" aria-checked=\\"false\\"><div class=\\"hs-toggle__track\\"><div class=\\"hs-toggle__handle\\"></div></div>default</label></hs-toggle>"`;

exports[`toggle renders should be toggle on when prop is present 1`] = `"<hs-toggle toggled=\\"\\" type=\\"success\\" class=\\"hydrated\\"><label class=\\"hs-toggle hs-toggle--success\\"><input type=\\"checkbox\\" aria-checked=\\"true\\"><div class=\\"hs-toggle__track\\"><div class=\\"hs-toggle__handle\\"></div></div>default</label></hs-toggle>"`;
